setup.cfg

```diff
@@ -116,10 +117,12 @@ install_requires =
     markdown>=2.5.2, <4.0
     markupsafe>=1.1.1, <2.0
     marshmallow-oneofschema>=2.0.1
+    mock<3.0.0  # required to keep apache-beam happy
     pandas>=0.17.1, <2.0
     pendulum~=2.0
     pep562~=1.0;python_version<"3.7"
     psutil>=4.2.0, <6.0.0
+    pyarrow<0.18.0
```

Comment:
This is quite old, I'm not sure we _want_ this globally.

Reply:
(Pyarrow is now at 2.0.0 for instance.)

Reply:
Yeah. I will experiment with moving those out of install_requires. I want to see if they will work for our tests first

Reply:
I only need to constraint deps in "install_requires" if they are already there transitively. I "eagerly" added them here to check if it works but likely httplib can be moved out

Reply:
Yep. All three are NOT transitive dependencies of airflow without extras, so It looks like we can move them where they belong.  Working on it.

Reply:
The only one that will have to stay there is dill, but I think this is not a problem